next/components/language-switcher.tsx

```diff
@@ -20,11 +21,17 @@ export function LanguageSwitcher() {
 
   // Close on click outside
   const ref = useOnClickOutside<HTMLUListElement>(close);
+  const { t } = useTranslation();
 
   return (
     <nav>
-      <button type="button" className="hover:underline" onClick={toggle}>
-        <span className="hidden sm:mr-2 sm:inline">
+      <button
+        type="button"
+        className="hover:underline"
+        onClick={toggle}
+        aria-label={t("language-switcher-button")}
+      >
+        <span className="sr-only sm:not-sr-only sm:mr-2 sm:inline">
           {languageLinks[locale].name}
         </span>
```

Comment:
(same question about aria-label duplication here)

Reply:
Actually I tested this part now 4 times and for some reason the first visible language option in the span is never read out loud by the voice over software so I need to pass that to the aria-label at least to read the current language in the button.